helix-query.yaml

```diff
@@ -0,0 +1,120 @@
+indices:
+  de:
+    include:
+      - /de/**
+    target: /de/query-index.json
+    properties:
+      lastModified:
+        select: none
+        value: parseTimestamp(headers["last-modified"], "ddd, DD MMM YYYY hh:mm:ss GMT")
+      image:
+        select: head > meta[property="og:image"]
+        value: match(attribute(el, "content"), "https:\/\/[^/]+(/.*)")
+      title:
+        select: head > meta[property="og:title"]
+        value: attribute(el, "content")
+      description:
```

Comment:
`<meta name="description" ...` not available. Checked `lib-franklin.js` > `export function setLanguage() {...` and it seems it will not be set